cupyx/scipy/stats/_morestats.py

```diff
@@ -0,0 +1,57 @@
+import cupy
+
+
+@cupy.fuse(kernel_name='find_sum')
+def find_sum(a):
+    return cupy.sum(a, axis=0)
+
+
+@cupy.fuse(kernel_name='find_log')
+def find_log(a):
+    return cupy.log(a)
+
+
+def boxcox_llf(lmb, data):
+    """The boxcox log-likelihood function.
+
+    Parameters
+    ----------
+    lmb : scalar
+        Parameter for Box-Cox transformation
+    data : array-like
+        Data to calculate Box-Cox log-likelihood for. If
+        `data` is multi-dimensional, the log-likelihood
+        is calculated along the first axis
+
+    Returns
+    -------
+    llf : float or cupy.ndarray
+        Box-Cox log-likelihood of `data` given `lmb`. A float
+        for 1-D `data`, an array otherwise
+
+    See Also
+    --------
+    scipy.stats.boxcox_llf
+
+    """
+
+    if data.ndim == 1 and data.dtype == cupy.float16:
+        data = data.astype(cupy.float64)
+    if data.ndim == 1 and data.dtype == cupy.float32:
+        data = data.astype(cupy.float64)
+    if data.ndim == 1 and data.dtype == cupy.complex64:
+        data = data.astype(cupy.complex128)
+
+    N = data.shape[0]
+    if N == 0:
+        return cupy.nan
+
+    logdata = find_log(data)
+
+    # Compute the variance of the transformed data
+    if lmb == 0:
+        variance = cupy.var(logdata, axis=0)
+    else:
+        variance = cupy.var(data**lmb / lmb, axis=0)
+
+    return (lmb - 1) * find_sum(logdata) - N/2 * find_log(variance)
```

Comment:
Can we combine these lines into a fused function? (`cupy.var` itself is not supported by CuPy fusion, so we'll implement it in the fused function for ourselves)

Reply:
Hi, @takagi! Thanks for pointing that out. 
Do you mean something like ...?
```python
@cupy.fuse(kernel_name='fused_func')
def fused_func(lmb, data):
    # the above lines
    return ...
```
If yes, then I don't think we can fuse these lines, because we cannot fuse `cupy.var`. I tried this and got the following:
```python
AttributeError: '_ArrayProxy' object has no attribute 'var'
```
Thanks

Reply:
ah, we need `keepdims=True` to compute variance in a fused function, and keepdims is currently not supported in CuPy fusion. I can't immediately answer that supporting keepdims in fused functions is easy or not.

Reply:
Sounds interesting to me!

Reply:
Sure, I have updated the PR. Thanks!